tests/scripts/basic-build-test.sh

```diff
@@ -74,6 +74,9 @@ make -j
 # Step 2 - Execute the tests
 TEST_OUTPUT=out_${PPID}
 cd tests
+if [ ! -f "seedfile" ]; then
+    dd if=/dev/urandom of="seedfile" bs=32 count=1
+fi
```

Comment:
`all.sh` needs a seedfile at `crypto/tests/seedfile`. Doesn't `basic-build-test.sh` also need a seedfile at `crypto/tests/seedfile` for `basic-build-test.sh` to work?

Reply:
No, because basic-build-test never runs crypto tests.

Reply:
That could be a problem, if the code coverage portion of that script is counting the duplicated aes.c in mbedtls as 0% coverage.

Reply:
Neither copy of `aes.c` is included in the coverage measure: `library/aes.c` is not built (as it should be, since it isn't used) and `crypto/library/aes.c` isn't measured (as it should be, since it's part of a separate project).

Reply:
OK, so our coverage tool doesn't look at any files that don't make it into the build for both what was tested as well as what wasn't tested. This is fine then.